x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/details/translations.ts

```diff
@@ -28,10 +28,10 @@ export const EXPERIMENTAL = i18n.translate(
   }
 );
 
-export const ACTIVATE_RULE = i18n.translate(
-  'xpack.securitySolution.detectionEngine.ruleDetails.activateRuleLabel',
+export const ACTIVATED_RULE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.ruleDetails.activatedRuleLabel',
   {
-    defaultMessage: 'Activate',
+    defaultMessage: 'Activated',
```

Comment:
Super nit: I wouldn't rename the key itself, or maybe I'd then rename the corresponding entries in `json` dictionaries as well. Just to not create additional work for the people who maintain the translations.